pyeudiw/trust/dynamic.py

```diff
@@ -161,13 +166,23 @@ def get_public_keys(self, issuer: Optional[str] = None, force_update: bool = Fal
         """
         trust_source = self._get_trust_source(issuer, force_update)
 
-        if not trust_source.keys:
+        keys = []
+        thumbprints = []
+
+        for handler in self.handlers:
+            if (key := trust_source.get_trust_param_by_handler_name(handler.__class__.__name__)) is not None:
+                for jwk in key.jwks:
+                    if key_from_jwk_dict(jwk) not in thumbprints:
+                        thumbprints.append(key_from_jwk_dict(jwk))
```

Comment:
```suggestion
                        thumbprints.append(key_from_jwk_dict(jwk).thumbprint("SHA-256"))
```

Reply:
This is already fixed here: [1a5a7d2](https://github.com/italia/eudi-wallet-it-python/pull/381/commits/1a5a7d2e7f36238ea915ba763934099039e70920)